KubeArmor/core/kubeUpdate.go

```diff
@@ -782,6 +814,45 @@ func (dm *KubeArmorDaemon) UpdateSecurityPolicy(action string, secPolicy tp.Secu
 	}
 }
 
+// UpdateDefaultPosture Function
+func (dm *KubeArmorDaemon) UpdateDefaultPosture(action string, namespace string, secPolicy tp.DefaultPosture) {
+	dm.EndPointsLock.Lock()
+	defer dm.EndPointsLock.Unlock()
+
+	dm.DefaultPosturesLock.Lock()
+	defer dm.DefaultPosturesLock.Unlock()
+
+	if action == "DELETED" {
+		delete(dm.DefaultPostures, namespace)
+	}
+
+	dm.DefaultPostures[namespace] = secPolicy
+
+	for idx, endPoint := range dm.EndPoints {
+		// update a security policy
+		if namespace == endPoint.NamespaceName {
+			if dm.EndPoints[idx].DefaultPosture == secPolicy && dm.DefaultPostures[namespace] == secPolicy {
+				continue
+			}
+
+			dm.Logger.UpdateDefaultPosture(action, namespace, secPolicy)
```

Comment:
is it possible to use dm.DefaultPostures instead of maintaining fd.DefaultPostures?

Reply:
dm construct doesn't seem to be available to feeder.